COMPILING.md

```diff
@@ -240,7 +240,11 @@ require manual modification of build files.
 2. Navigate to the *top level* folder of the project. This is different from
    the Makefile build, which requires you to navigate to the `src` directory
    first.
-3. Generate build files with CMake:
+3. Update git submodules:
+   ```
+   git submodule update --init --recursive
```

Comment:
` --recursive` is not needed

Reply:
While I agree, this should really propagate to not use this in `make setup-submodules` either.